### DISCUSSION TITLE
Does anyone know how to implement nested sub-tasks?

### DISCUSSION THREAD

**Discussion by jeffreyouni** [kanboard/kanboard#5123] [Q&A] [answered]
Hi guys,

Does anyone know how to implement nested subtasks?

For example, a sub-subtask under a subtask, and inside the sub-subtask, we can still assign anyone to do it.

I need this feature because I need person A to finish subtask A, and person A can create a subtask B under subtask A to person B, and so on.

Does anyone know how to do this or if there's any similar way we can do this?

Thank you so much for you help!

**Comment by aljawaid** [comment]
> Hi guys,
> 
> Does anyone know how to implement nested subtasks?
> 
> For example, a sub-subtask under a subtask, and inside the sub-subtask, we can still assign anyone to do it.
> 
> I need this feature because I need person A to finish subtask A, and person A can create a subtask B under subtask A to person B, and so on.
> 
> Does anyone know how to do this or if there's any similar way we can do this?
> 
> Thank you so much for you help!

I think you may be complicating it on your part. Stick to tasks.

- Task A needs task B and task C to complete
- User 1 is assigned to task A and task B.... each task and subtask can be completed by user 1
- User 2 is left to deal with task C

looking at task A, you can see that it is linked to other tasks/subtasks.... go into each task to see the subtasks

**Comment by fguillot** [comment, marked as answer]
> Does anyone know how to implement nested subtasks?

Kanboard does not support nested subtasks. However, you can have relationships between tasks.
